responses/__init__.py

```diff
@@ -668,6 +668,9 @@ def __exit__(self, type, value, traceback):
     def activate(self, func):
         return get_wrapped(func, self)
 
+    def calls_for_match(self, match):
+        return len([call for call in self.calls if call.request.url == match.url])
```

Comment:
Should this also take into account the other kinds of matchers that responses has? For example encoded body and querystring matching?

Reply:
I'd suspect no but it could

Reply:
fixed @markstory